src/server.py

```diff
@@ -26,8 +26,8 @@ def create_self_signed_cert(cert_dir, cert_file, key_file, hostname):
     # from https://gist.github.com/ril3y/1165038
     cert_path = realpath(join(cert_dir, cert_file))
     key_path = realpath(join(cert_dir, key_file))
-    if not exists(cert_path) or not exists(key_path):
 
+    if not exists(cert_path) or not exists(key_path):
```

Comment:
If you're calling a function called `create_self_signed_cert` shouldn't we always create these files?

Reply:
Good point.